### DISCUSSION TITLE
Welcome to generator-jhipster-micronaut Discussions!

### DISCUSSION THREAD

**Discussion by mraible** [jhipster/generator-jhipster-micronaut#328] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by javagrails** [comment]
I use java 21 and node 20.6.x or 18.x.x but below error shows here with the latest version 3.1.0
ERROR! An error occured while running jhipster-micronaut:server#writeFiles
ERROR! ERROR! Template file src/main/java/package/config/dbmigrations/InitialSetupMigration.kt was not found at /Users/mypc/.nvm/versions/node/v20.9.0/lib/node_modules/generator-jhipster-micronaut/generators/server/templates

**Reply by mraible** [reply]
If I use Java 21 and Node 20.12.0, I can run `mhipster --defaults` and everything completes just fine.

<img width="1442" alt="Screenshot 2024-03-27 at 11 06 22 PM" src="https://github.com/jhipster/generator-jhipster-micronaut/assets/17892/0abff0d1-9df4-45cc-888a-eb99668411c0">

<img width="1442" alt="Screenshot 2024-03-27 at 11 07 13 PM" src="https://github.com/jhipster/generator-jhipster-micronaut/assets/17892/c56bfb1c-7317-466b-8681-d076e8070d3f">

Please provide the steps about how to reproduce the issue you're having.

**Reply by javagrails** [reply]
❯ java --version
openjdk 21 2023-09-19
OpenJDK Runtime Environment (build 21+35-2513)
OpenJDK 64-Bit Server VM (build 21+35-2513, mixed mode, sharing)

❯ node --version
v20.9.0

❯ jhipster --version
8.2.1

❯ npm install -g generator-jhipster-micronaut

❯ mhipster --version
INFO! No custom commands found within blueprint: generator-jhipster-micronaut at /Users/salepc/.nvm/versions/node/v20.9.0/lib/node_modules/generator-jhipster-micronaut
3.1.0 (generator-jhipster 8.2.1)

**❯ mhipster --defaults** works fine as you show in your screen shoots but below two command not working with the same error it stops in the given screen shoots 
**❯ mhipster** OR **❯ jhipster --blueprints micronaut**

<img width="1700" alt="mh-prob-01" src="https://github.com/jhipster/generator-jhipster-micronaut/assets/3772814/d1eb31b9-792b-4f4c-92a1-23ad55f89312">

<img width="1752" alt="mh-prob-02" src="https://github.com/jhipster/generator-jhipster-micronaut/assets/3772814/fdd4de73-5af0-406b-ab74-49b0ead3b41f">

Thanks - Matt Raible

**Comment by mraible** [comment]
The prompt to use Spring WebFlux shouldn't exist, so that's definitely concerning. Can you please enter an issue so we can try to get this fixed?

**Reply by javagrails** [reply]
sure & why not I want to contribute here jhipster-micronaut as well as core-jhipster and jhipster-quarkus. let me open an issue and instead of WebFlux what we use in micronaut?

ISSUE https://github.com/jhipster/generator-jhipster-micronaut/issues/329

**Reply by javagrails** [reply]
I found two files 
#### https://github.com/jhipster/generator-jhipster/blob/main/generators/server/prompts.js
#### https://github.com/jhipster/generator-jhipster-micronaut/blob/main/generators/server/prompts.cjs
first link has the text [message: 'Do you want to make it reactive with Spring WebFlux?'] but can not understand how logic comes in 2nd file

**Reply by mraible** [reply]
Everything should be fixed now in the v3.3.0 release. Can you please try again?
